[AGW] mconfig: pipelineD: Allow vlan networking for AGW management iface

pshelar opened the pull request:
## Summary

AGW operator would like to configure vlan tag for SGi interface
for management interface. This mconfig could allow them to
configure it via orc8r API

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

<!--
    Tag your PR title with the components that it touches.
    E.g. "[lte][agw] Changeset" or "[orc8r][docker] ..."
-->


<!-- Enumerate changes you made and why you made them -->

## Test Plan
`go test builder_servicer_test.go`

<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

xjtian commented on lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml:
```diff
@@ -1677,6 +1677,12 @@ definitions:
         example: '8.8.4.4'
+      sgi_management_iface_vlan:
+        type: string
+        description: This set specific vlan id for management interface traffic of the AGW.
```
"VLAN ID for management interface traffic on the AGW"

xjtian reviewed (approved):
Merge after updating the description

pshelar committed:
[AGW] mconfig: pipelineD: Allow vlan networking for AGW management iface

AGW operator would like to configure vlan tag for SGi interface
for management interface. This mconfig could allow them to
configure it via orc8r API

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>
